src/less/progress-stepper/base/progress-stepper.less

```diff
@@ -183,4 +191,12 @@ hr.progress-stepper__separator--upcoming,
         font-size: @font-size-14;
         width: @progress-stepper-large-min-width;
     }
+
+    .progress-stepper__text h2,
```

Comment:
Is this a good candidate for `:is`?

```css
.progress-stepper__text :is(h2, h3, h4, h5, h6);
```

https://developer.mozilla.org/en-US/docs/Web/CSS/:is

Reply:
We can. 
However it seems like a recent change and only the latest browsers support it like safari 14. 
We should revisit when our browser support changes imo.

Reply:
Argh, yeah. I'm forgetting we are stuck on Safari 10 :-( Thanks!